internal/cmd/util.go

```diff
@@ -72,6 +74,12 @@ func firstNonEmptyString(ss ...string) string {
 	return ""
 }
 
+// newCacheHTTPClient returns a new http.Client that will cache responses
+// according to the HTTP RFC.
+func newCacheHTTPClient(cacheDirAbsPath string) *http.Client {
```

Comment:
nit: Through the code variables with the type `chezmoi.AbsPath` the name `*AbsPath`. Please change the type of the argument from `string` to `chezmoi.AbsPath`. The reason for using `AbsPath` instead of `string` is to improve type safety through the code. There's a bit more explanation in the [architecture guide](https://github.com/twpayne/chezmoi/blob/master/docs/ARCHITECTURE.md#path-handling).